Plots/MapProjections/NCL_lcnative_1_lg.py

```diff
@@ -0,0 +1,73 @@
+"""
+
+NCL_lcnative_1_lg.py
+================
+This script illustrates the following concepts:
+    - Drawing contours over a map using a native lat,lon grid
+    - Drawing filled contours over a Lambert Conformal map
+    - Zooming in on a particular area on a Lambert Conformal map
+    - Subsetting a color map
+    - Using all three Cartopy Lambert projections to find the best fit for visualization
+    - Implementing best practices for choosing contour color scheme
+    
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/lcnative_1.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/lcnative_1_lg.png
+
+"""
+###############################################################################
+# Import packages:
+
+import numpy as np
+import xarray as xr
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import geocat.datafiles as gdf
+import matplotlib.ticker as mticker
+from geocat.viz import cmaps as gvcmaps
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/pre.8912.mon.nc"), decode_times=False)
+
+# Extract a slice of the data
+t = ds.pre.isel(time=0)
+
+lat2d = ds.lat[:,:]
+lon2d = ds.lon[:,:]
+
+
+###############################################################################
+# Plot:
+
+def Plot(proj, row, col, pos, title):
+
+    plt.figure(figsize=(14, 14))
+    # Generate axes using Cartopy and draw coastlines
+    projection = proj#LambertAzimuthalEqualArea()#central_longitude=45)#, standard_parallels=(36,55), globe=ccrs.Globe())
+    ax = plt.subplot(row, col, pos, projection=projection)
+    ax.set_extent((28, 57, 20, 47), crs=ccrs.PlateCarree())
+    ax.coastlines(linewidth=0.5)
+    
+    gl = ax.gridlines(draw_labels=True, dms=False, x_inline=False, y_inline=False)
+    gl.xlabels_top = False
+    gl.ylabels_right = False
```

Comment:
The NCL example has latitude and longitude labels on the top and right of the plot. I know that the labels don't look quite right when you have tried to add them in before so I think it's okay that they aren't included. Maybe we should add a comment here mentioning Python's limitation with these labels for certain projections.

Reply:
I think it may be beneficial to actually turn them on but use your idea of having a comment here about the discrepancies  of Python/NCL with the Lambert Conformal projection. This would also make sense just overall so that it doesn't seem random that we have two extra projections that aren't Lambert Conformal for this example as well.